main.js

```diff
@@ -241,87 +284,56 @@ function createPopup(index) {
   });
 }
 
-for (let i = 0; i < projectsArray.length; i += 1) {
-  const projectWrapper = document.getElementsByClassName('project_wrapper')[0];
-
-  const projectCard = document.createElement('article');
-  projectCard.classList.add('project_card', `card${i}`);
-
-  const projectCardImage = document.createElement('img');
-  projectCardImage.classList.add('project_image');
-  projectCardImage.setAttribute('src', projectsArray[i].imgMobile);
-  projectCardImage.setAttribute('alt', projectsArray[i].name);
-  projectCard.appendChild(projectCardImage);
-
-  const projectCardWrapper = document.createElement('div');
-  projectCardWrapper.classList.add('project_card_wrapper');
-  projectCard.appendChild(projectCardWrapper);
-
-  const projectCardTitle = document.createElement('h4');
-  projectCardTitle.classList.add('project_card_title');
-  projectCardTitle.innerText = projectsArray[i].name;
-  projectCardWrapper.appendChild(projectCardTitle);
-
-  const projectCardTags = document.createElement('ul');
-  projectCardTags.classList.add('project_card_tags');
-  projectCardWrapper.appendChild(projectCardTags);
-
-  for (let j = 0; j < projectsArray[i].techtags.length; j += 1) {
-    const projectCardTag = document.createElement('li');
-    projectCardTag.innerText = projectsArray[i].techtags[j];
-    projectCardTags.appendChild(projectCardTag);
+document.addEventListener('DOMContentLoaded', () => {
+  const projectCardButtons = document.querySelectorAll('.project_card_button');
+
+  for (let i = 0; i < projectCardButtons.length; i += 1) {
+    ((index) => {
+      projectCardButtons[index].addEventListener('click', () => {
+        createPopup(index);
+        const popupWindow = document.getElementsByClassName('popup__window')[index];
+        popupWindow.style.display = 'block';
+      });
+    })(i);
   }
+});
 
-  const projectCardButton = document.createElement('button');
-  projectCardButton.classList.add('project_card_button');
-  projectCardButton.setAttribute('type', 'button');
-  projectCardButton.innerText = 'See Project';
-  projectCardWrapper.appendChild(projectCardButton);
+// Form Validation //
 
-  projectWrapper.appendChild(projectCard);
+/* ------- SAVE INPUT VALUES TO LOCAL STORAGE -------*/
+const contactForm = document.querySelector('.contact__form');
 
-  projectCardButton.addEventListener('click', () => createPopup(i));
-}
+const saveFormData = () => {
+  const formData = {
+    fullName: document.getElementById('fullname').value,
+    email: document.getElementById('email').value,
+    message: document.getElementById('message').value,
```

Comment:
- [ ] Nice work storing the form data in a single javascript object as suggested in previous review :clap: However the new change that you made insert bug in your code. Kindly notice that the message input `id` is `text` not `message`. That will fix the bug the data will be saved.